modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -26,5 +24,13 @@ class HigherLevelReviewsController < AppealsApi::V2::DecisionReviews::HigherLeve
     def token_validation_api_key
       Settings.dig(:modules_appeals_api, :token_validation, :higher_level_reviews, :api_key)
     end
```

Comment:
Of course, storing schemas based on the API name and version requires the API name and version to be known. 

To accomplish this in all the controllers, I moved the instantiation of the `AppealsApi::FormSchemas` object (which happens at least once in every decision reviews/segmented controller) into its own method in all cases, and passed in the name and version there. Incidentally this also allowed me to remove the `SCHEMA_VERSION` class constants (since they were only used to instantiate this object)